src/main/resources/templates/entries.html

```diff
@@ -6,6 +6,9 @@
     <header th:replace="fragments/global-header.html::global-header"></header>
 
     <main id="wrapper" role="main">
+      <th:block th:if="${register.phase == 'beta'}">
+        <div th:replace="fragments/register-intro.html::register-intro"></div>
```

Comment:
Is it by design to only have this bit for beta registers?

Reply:
Yes